web/docs/use/run/README.md

```diff
@@ -57,3 +57,11 @@ To select a specific VAST server to connect to,
 `--endpoint=host:port` on the command line, exporting the environment variable
 `VAST_ENDPOINT=host:port`, or setting the configuration option
 `vast.endpoint: host:port` in your `vast.yaml`.
+
+## Client connection failure
+
+In the event of a connection failure, the clients will attempt to connect up to
+100 times with a 3-second delay between each attempt.
```

Comment:
This is still limited to `vast.connection-timeout`, right? So by default it will be at most four attempts: at 0s, 3s, 6s, and 9s, and then the fourth attempt will be cancelled after 1s.

Do the defaults make sense with this in mind?

Reply:
@patszt and I discussed that we want the initial time period to cover 5mins / 300secs. This is where this 100 * 3 split comes from.

Reply:
How is this supposed to interact with `vast.connection-timeout`? For example, we advise users to set `VAST_CONNECTION_TIMEOUT=0s` when they want clients to use an infinite timeout. Shouldn't we be re-using that configuration option such that users can still set the total?

Reply:
vast.connection-timeout is used as a timeout between middleman and the connector actor. At least in a blocking connect_to_node.cpp.

This means that a 3 second delay is started only after connection failure. The duration of connection process is not taken into account.
Maybe the usage of vast.connection-timeout as middleman timeout request is bad in this case.

What do you think about having another parameter that models this timeout?
For now it could be a hardcoded value just like the retry delay and retry amount.

I would change vast.connection-timeout to vast.node-request-timeout
Use vast.connection-timeout as a middleman request timeout